### DISCUSSION TITLE
Import a solution gives "An unexpected error occurred"

### DISCUSSION THREAD

**Discussion by Joseff-Evans** [microsoft/powerplatform-build-tools#859] [Q&A] [answered]
Hey,

I have a pipeline in devops that uses the build tools and when importing to one of our environments we got:

The reason given was: An unexpected error occurred.
Microsoft PowerPlatform CLI
Version: 1.31.6+g9147a23
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

When checking the environment solution history I can see the actual error and its details - is there a way to get this to feed back to my pipeline in any way or are there any suggestions on linking the failed run to a history item to get better error details?

**Comment by petrochuk** [comment, marked as answer]
You can check "Enable system diagnostics" to get more detailed logging from pipeline and build tools
![image](https://github.com/microsoft/powerplatform-build-tools/assets/30735471/a02fa1e4-de86-49d6-b0e6-b346e322f390)

**Reply by Joseff-Evans** [reply]
Thanks for the info - will try it out and see what we get the next time we have a failed run.
